pkg/jsonchecker/jsonchecker.go

```diff
@@ -15,15 +15,14 @@ import (
 	"github.com/cilium/tetragon/api/v1/tetragon"
 	ec "github.com/cilium/tetragon/api/v1/tetragon/codegen/eventchecker"
 	"github.com/cilium/tetragon/api/v1/tetragon/codegen/helpers"
-	"github.com/cilium/tetragon/pkg/eventcache"
 	"github.com/cilium/tetragon/pkg/logger"
 	"github.com/cilium/tetragon/pkg/testutils"
 	"github.com/sirupsen/logrus"
 )
 
-var (
+const (
 	Retries    = 13
-	RetryDelay = eventcache.EventRetryTimer + (1 * time.Second)
+	RetryDelay = 2 * time.Second
```

Comment:
This is a slight change from before where it was 3 seconds. Is this expected?

Reply:
Missed the `+1`. I have made that 3 again. I would say that as this is a test and the test passes, this was not a big deal, but I changed that to be on the safe side.